.github/workflows/flow-linux-platforms.yml

```diff
@@ -92,6 +108,8 @@ jobs:
     secrets: inherit
     with:
       get-redis: ${{ inputs.redis-ref }}
+      rejson: ${{ (inputs.rejson == true || inputs.rejson == 1) && 1 || 0 }}
```

Comment:
Is this tested? (for resulting in `1` or `0` for any `inputs.rejson` value)

Reply:
yes. I did it because I got an error before :smiling_face_with_tear: